release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -555,6 +555,12 @@ To manage the cloud controller manager and cloud node manager deployments and li
 
 For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_red-hat-operators[Cluster Cloud Controller Manager Operator] entry in the _Red Hat Operators reference_.
 
+[discrete]
+[id="ocp-4-9-canary-rollout-update-changes"]
+==== Performing a canary rollout update
+
+With {product-title} 4.9, the process to perform a canary rollout update has changed. For a detailed overview of this process, see xref:../updating/update-using-custom-machine-config-pools.html#update-using-custom-machine-config-pools-about_update-using-custom-machine-config-pools[About performing a canary rollout update]. 
```

Comment:
`.html` -> `.adoc` in the xref suffix

Reply:
> the process to perform a canary rollout update has changed

I think this is our 1st time to introduce canary rollout update in our official doc, but not change some existing ones, right?